src/World.h

```diff
@@ -1051,13 +1049,13 @@ class cWorld :
 	cWorld(const AString & a_WorldName, eDimension a_Dimension = dimOverworld, const AString & a_OverworldName = "");
 	virtual ~cWorld();
 
-	void Tick(float a_Dt, int a_LastTickDurationMSec);
+	void Tick(std::chrono::milliseconds a_Dt, std::chrono::milliseconds a_LastTickDurationMSec);
```

Comment:
Now the real question: What's the difference between `a_Dt` and `a_LastTickDurationMSec`?